frontend/src/components/Forms/CreateCourseDescription.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+import { Button, Form, Input, Title, Wrapper } from '../GlobalStyle'
+
+export default function CreateCourseDescription({ onAddDescription }) {
+  const [updatedCourseDescription, setUpdatedCourseDescription] = useState('')
+
+  const handleSubmit = (event) => {
```

Comment:
just nitpicking, but maybe rename to ``updateCourseDescription`` instead of a general name?